### PR TITLE
Update mescheder2018occupancy_1.md

### PR DESCRIPTION
**Pull request by ishanashah**

### CODE REVIEW

**Comment by ishanashah** [comment]
@kelseyball Hi, my summary is ready for review.

**Comment by ishanashah** [comment]
Sorry about that. @elampietti, my summary is ready to review.

File: _summaries/mescheder2018occupancy_1.md [hunk 1]
```diff
@@ -1,19 +1,66 @@
...
+
+## Related Work
+* Voxels: 3D pixels, but memory requirements grow cubically with size.
+* Point Clouds: Don't have connectivity information between points, so difficult to extract surfaces.
+* Meshes: Usually based on a template mesh, because it is very difficult to handle meshes of very different objects.
+
+![FIGURE_1](mescheder2018occupancy_1a.PNG)
+
+## Solution
+Reason about occupancy not only at fixed discrete 3D locations, but at any possible 3D point.
```
Comment (elampietti) [inline _summaries/mescheder2018occupancy_1.md:20]: Mention how this results in reduced memory usage during training

File: _summaries/mescheder2018occupancy_1.md [hunk 2]
```diff
@@ -1,19 +1,66 @@
...
+
+The function is called an occupancy network, because it predicts the probability that a point is occupied.
+
+The surface of points where the probability of occupancy is 0.5 is the surface of the object.
+
+For 3D reconstruction, the occupancy network takes a pair of the input image and 3D location as inputs.
+
+## Training and Inference
+
+Occupancy network is trained on a random sample of points with known labels.
```
Comment (elampietti) [inline _summaries/mescheder2018occupancy_1.md:32]: Could add some details on the occupancy network implementation details: (fully connected network with 5 ResNet blocks conditioned on the input using conditional batch normalization).

File: _summaries/mescheder2018occupancy_1.md [hunk 3]
```diff
@@ -1,19 +1,66 @@
...
+
+Condition Occupancy Networks on Noisy Point Clouds:
+
+![FIGURE_4](mescheder2018occupancy_1d.PNG)
+
+
+Condition Occupancy Networks on Low-Resolution Voxels (3D Super-resolution):
+
+![FIGURE_5](mescheder2018occupancy_1e.PNG)
+
```
Comment (elampietti) [inline _summaries/mescheder2018occupancy_1.md:61]: Could add some details on their ablation study such as the effect of the sampling strategy on final performance.

File: _summaries/mescheder2018occupancy_1.md [hunk 4]
```diff
@@ -1,19 +1,66 @@
...
+Evaluate on ShapeNet dataset against voxel-based, point-based, and mesh-based approaches.
+
+Representation Power:
+* Evaluate on the "chair" category of the ShapeNet dataset.
+* The occupancy network is able to encode all 4746 training samples with only 6M parameters.
+
+![FIGURE_3](mescheder2018occupancy_1c.PNG)
+
+Condition Occupancy Networks on Noisy Point Clouds:
```
Comment (elampietti) [inline _summaries/mescheder2018occupancy_1.md:53]: Could add some more details on how these results were obtained.

**Pull request merged**